handbook/engineering/hiring/engineering-manager-backend-platform.md

```diff
@@ -0,0 +1,68 @@
+# Engineering Manager - Backend Platform
+
+We are looking for an engineering manager to join the team that is building the backend platform upon which product engineering teams can iterate quickly and safely. The team's mission is to promote and champion consistency and simplicity across all backend code by building shared tools, libraries and infrastructure for common use cases, scale and maintain said infrastructure and ultimately create leverage for and accelerate other teams.
+
+The [Cloud team](https://about.sourcegraph.com/handbook/engineering/cloud) team will become the Backend Platform team once we hire a Product Manager to drive the Cloud project forward.
+
+## Responsibilities
+
+You will:
+
+- Facilitate a inclusive and cohesive team environment where everyone is happy, motivated, and productive.
+- Model, teach, and reinforce [our company values](../../company/values.md) and [our guiding engineering principles](index.md#guiding-principles) to the team.
+- Ensure the team has clear incremental goals that are documented and are always up-to-date.
+- Regularly communicate the team's progress toward their goals as well as changes in team goals to appropriate stakeholders.
+- Support and coach teammates to grow in their careers and fulfill their responsibilities.
+- Grow the team in a sustainable way so that the team can accomplish more over time.
+
+## Qualifications
+
+- You have 2+ years of experience directly managing at least 6 full-time software engineers. We expect you already know how to lead a team.
```

Comment:
What do you think about dropping the minimum to 4? (1) It is just a minimum qualification and (2) it feels like there isn't much material difference between 4 and 6.

```suggestion
- You have 2+ years of experience directly managing at least 4 full-time software engineers. We expect you already know how to lead a team.
```